hs/spec_compliance/BUILD.bazel

```diff
@@ -72,6 +73,12 @@ STACK_DEPS = [
     "@stackage//:zlib",
 ]
 
+disable_hermetic_cc_binary(
```

Comment:
should we just make this one the default?  we can rename this one to `ic-ref-test` and have the private one be `ic-ref-test-FOO` to avoid confusion